builder/test/softmax.cc

```diff
@@ -199,6 +200,17 @@ TEST_P(softmax, pipeline) {
     ASSERT_THAT(out_b, testing::Pointwise(testing::FloatNear(1e-6f), ref_b));
   }
 
+  if (split_b > 0) {
+    ASSERT_EQ(eval_ctx.heap.total_size, (split_b * D * 2 +
```

Comment:
I think we should break this into separate declarations, one for each buffer, so this expression is the sum of 5 variables. Right now it's pretty hard to understand.